sloprovisioning.go

```diff
@@ -2051,3 +2052,24 @@ func (c *Client) UpdateHostGroupHosts(ctx context.Context, symID string, hostGro
 	}
 	return updatedHostGroup, nil
 }
+
+// GetPortListForSymmetrix returns a list of all the ports on a specified Symmetrix array.
```

Comment:
Comment is incorrect since it's filtering by protocol and not returning all ports

Reply:
Updated the comment to
`// GetPortListByProtocol returns a list of ports associated with a given protocol for a specified Symmetrix array.`